statsmodels/tsa/arima/estimators/gls.py

```diff
@@ -240,6 +240,16 @@ def gls(endog, exog=None, order=(0, 0, 0), seasonal_order=(0, 0, 0, 0),
         # argument applies a Prais-Winsten-type normalization to the first few
         # observations to ensure homoskedasticity). Brockwell and Davis
         # mention that they also take this approach in practice.
+
+        # GH-6540: AR must be stationary
+
+        if ar_params.shape[0] and \
```

Comment:
I forgot, you can just use `if not p_arma.is_stationary:` here.

Reply:
Thanks.